### ISSUE TITLE
InteroStart fails to identify stack.yaml

### ISSUE THREAD

**Issue by dwaldhalm** [parsonsmatt/intero-neovim#104]
On Windows 8
InteroStart
Failed to identify stack.yaml, does it exist?

I have a stack project (default by 'stack new <name>'.  It compiles and runs in the repl from cmder, but the Intero plugin doesn't see the stack.yaml..

**Comment by parsonsmatt** [comment]
Can you post the output of executing `:! stack path --config-location` in your Neovim?

**Comment by dwaldhalm** [comment]
C:\sr\global-project\stack.yaml

**Comment by parsonsmatt** [comment]
That's telling me that your `neovim` present working directory isn't in the directory of the file, or one of the contained directories. Can you try doing `:cd path/to/your/haskell/project` and retrying?

**Comment by dwaldhalm** [comment]
Yes.  After changing the working directory, I see the local stack.yaml, but still get the same error.

**Comment by parsonsmatt** [comment]
Cool, I've identified the problem. Line 75 in `autoload/intero/process.vim`:

```vim
            call filter(l:stack_path_config, "v:val =~? '^/.*\.yaml'")
```

This regex doesn't match the output for Windows file systems. We'll need to modify the regex to make it work.

Thanks for reporting, I should have this fixed up soon :)

**Comment by parsonsmatt** [comment]
I've pushed a fix in 4058f08047eb08d6558d22ae8d748db7d8e20299. Can you pull latest master and try again?

**Comment by dwaldhalm** [comment]
That works.  Thanks!
